System.IO.FileSystem/tests/System.IO.FileSystem.Tests/nano.runsettings

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RunSettings>
+  <!-- Configurations that affect the Test Framework -->
+  <RunConfiguration>
+    <MaxCpuCount>1</MaxCpuCount>
+    <ResultsDirectory>.\TestResults</ResultsDirectory>
+    <!-- Path relative to solution directory -->
+    <TestSessionTimeout>120000</TestSessionTimeout>
+    <!-- Milliseconds -->
+    <TargetFrameworkVersion>Framework40</TargetFrameworkVersion>
+  </RunConfiguration>
+  <nanoFrameworkAdapter>
+    <Logging>None</Logging>
+    <IsRealHardware>False</IsRealHardware>
```

Comment:
are the tests running in the virtual device? Is the System.IO.File having a native implementation supported there?

Reply:
Currently the tests are just checking that the enumerations are correct (in theory) aligned to the full fat .NET implementation. We need to make sure these can align with the native etc..